Dockerfile.kvdb

```diff
@@ -15,9 +14,9 @@ RUN apt-get update && \
   apt-get -y install unzip curl make git default-jre
 
 
-RUN curl -s -L https://dl.google.com/go/go1.10.7.linux-amd64.tar.gz | tar -C /usr/local/ -xz  && \
-  curl -s -L https://github.com/coreos/etcd/releases/download/v3.2.15/etcd-v3.2.15-linux-amd64.tar.gz -o /tmp/etcd-v3.2.15-linux-amd64.tar.gz  && \
-  mkdir -p /tmp/test-etcd && tar xzvf /tmp/etcd-v3.2.15-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1 && cp /tmp/test-etcd/etcd /usr/local/bin  && \
+RUN curl -s -L https://dl.google.com/go/go1.17.13.linux-amd64.tar.gz | tar -C /usr/local/ -xz  && \
+  curl -s -L https://github.com/coreos/etcd/releases/download/v3.4.24/etcd-v3.4.24-linux-amd64.tar.gz -o /tmp/etcd-v3.4.24-linux-amd64.tar.gz  && \
+  mkdir -p /tmp/test-etcd && tar xzvf /tmp/etcd-v3.4.24-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1 && cp /tmp/test-etcd/etcd /usr/local/bin  && \
```

Comment:
I'd recommend using one-liners for both:

```
RUN curl -fsSL --retry 5 --retry-max-time 60 https://dl.google.com/go/go1.17.13.linux-amd64.tar.gz | tar -C /usr/local/ -xz  && \
    curl -fsSL --retry 5 --retry-max-time 60 https://github.com/coreos/etcd/releases/download/v3.4.24/etcd-v3.4.24-linux-amd64.tar.gz | tar xzvf -C /usr/local/bin/ --strip-components=1 -xzvf && \
    ...
```

Reply:
Unlike go, when you unzip the etcd tar.gz file, you get a bunch of stuff. I am guessing that the original author did not want to dump all that into /usr/local/bin. That's why they used a tmp dir for the initial unzip. Also, this container is not built with every make. It is built and pushed to the docker repo one time. So, I am leaving this as it is for now.